developer/contributing/pull-requests.rst

```diff
@@ -60,12 +67,11 @@ You can copy and paste the following template:
 
     __informations:__
 
-    | q             | a
-    | ------------- | ---
-    | tests pass?   | [yes|no => <why?>]
-    | fixed tickets | [<which?>|none]
-    | bc breaks     | [description]
-    | doc           | [url to commit|none]
+    | q                | a
+    | ---------------- | ---
+    | Fixed tickets    | [<which?>|none]
+    | BC breaks        | [description]
+    | Documentation PR | [url to PR|none]
```

Comment:
Maybe leave the tokens out? Would make copy + paste quicker (no need to highlight and delete the holding texts). The example below shows how it should be done.